sdk-base/src/main/java/com/mapbox/maps/plugin/annotation/ClusterOptions.kt

```diff
@@ -0,0 +1,69 @@
+package com.mapbox.maps.plugin.annotation
+
+import android.graphics.Color
+
+/**
+ * Options to show and configure symbol clustering with using SymbolManager.
+ * <p>
+ * It exposes a minimal of configuration options, a more advanced setup can be created manually with
+ * using CircleLayer and SymbolLayers directly.
+ * </p>
+ */
+data class ClusterOptions(
+
+  /**
+   * If the data is a collection of point features, setting this to true clusters the points
+   * by radius into groups. Cluster groups become new `Point` features in the source with additional properties:
+   * - `cluster` Is `true` if the point is a cluster
+   * - `cluster_id` A unqiue id for the cluster to be used in conjunction with the
+   * [cluster inspection methods](https://www.mapbox.com/mapbox-gl-js/api/#geojsonsource#getclusterexpansionzoom)
+   * - `point_count` Number of original points grouped into this cluster
+   * - `point_count_abbreviated` An abbreviated point count
+   */
+  val cluster: Boolean = true,
```

Comment:
Where do those defaults come from? Shouldn't it go from style spec?

Reply:
If users create and set the `ClusterOptions` instance, that means they want to use the cluster feature, so set the default value to true will be more user-friendly.

Reply:
I am more asking about all other concrete values below.

Reply:
There is no style-spec default way to style clusters themselves, we need to come up with our own values but might need to align on this with iOS